src/levelGenerator.cpp

```diff
@@ -52,6 +55,13 @@ void LevelGenerator::onFixedUpdate()
     const float despawn_range = 300;
     sp::Vector2f camera_position = sp::Vector2f(getScene()->getCamera()->getPosition2D());
 
+    if (fixed_frame_count > 0)  // Work around the fact that the first frame all buttons seem pressed
+    {
+         for(auto player : players) {
```

Comment:
codestyle

Reply:
Oops. Should have paid more attention to style. Will change.